docs/DeveloperGuide.md

```diff
@@ -65,6 +65,20 @@ their distinct file paths.
 
 ### Command handling component
 
+![commandHandling.png](./Diagrams/commandHandling.png)
```

Comment:
It may also be better to have something on all the line to standardise? E.g. `XYZListCommand()` for the constructor and `return` for void return functions.

Reply:
I have created new directories in `Diagrams` for better organization, so it may also be better to move the diagrams into the `Diagrams/Commands` directory.